WordPress/gutenberg
# [Mobile] Modified More block to be read-only

gutenberg-mobile side PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1079

As mentioned in https://github.com/wordpress-mobile/gutenberg-mobile/issues/859#issuecomment-498321249, I opted for modifying the `More` block to be read only, in order for it to be ready for the next release.

I did a fast test using RichText. It does work well on iOS, but it had some alignment issues on Android, that would need native code to be fixed. It also had a visual glitch on split.

Fixing those issues on Android, it looks promising to use RichText to make this component editable.
Alternatively, we might also want to think on a simpler PlaneText component that we can manipulate at the native level for these cases.

On the design side, I noticed on web that More block and Page Break block look the same, so I implemented the same design we use for Page Break on mobile too.

![more-block](https://user-images.githubusercontent.com/9772967/58960670-ee5ef700-87a7-11e9-8d07-1879c69c80ce.png)

To test:
- Run the app.
- Search for the Read More block in the example content (or add one if it's a new post)
- Check that it's not editable.
- Check that it can be selectable by tapping anywhere in the block.
- Paste this in the HTML mode:
```



```
- Go back to Visual mode.
- Check that it shows custom content.
